src/components/Person/Person.jsx

```diff
@@ -1 +1,39 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const personAge = (
+    <p className="Person__age">
+      {`I am ${person.age}`}
+    </p>
+  );
+  const notMarried = <p className="Person__partner">I am not married</p>;
+  const myWife = (
+    <p className="Person__partner">
+      {person.partnerName}
+      {' '}
+      is my wife
+    </p>
+  );
+  const myHusband = (
+    <p className="Person__partner">
+      {person.partnerName}
+      {' '}
+      is my husband
+    </p>
+  );
```

Comment:
Please don't mix content with the layout. Here you need to get values and insert them into JSX inside the return. Now your code has a lot of repeating parts.
- `personAge` is useless, move the element back to the structure;
- replace `notMarried`, `myWife` and `myHusband` with: `partnerStatus` and `marriageStatus` with only content values.

```js
const partnerStatus = person.sex === 'm'
  ? 'option1'
  : 'option2';
const marriageStatus = person.isMarried
  ? `text with ${partnerStatus}`
  : 'not married';
```